### ISSUE TITLE
Higher Learning Rate

### ISSUE THREAD

**Issue by pathak22** [jazzsaxmafia/Impainting#2]
Hi Taeksoo,

I suggest using higher learning for training. In the paper, I used following settings:
LR = 0.002 for generator
LR = 0.0002 for discriminator
I believe it should improve the quality of results a bit.  :) Curios to see the updated results.

Thanks,
Deepak

**Comment by jazzsaxmafia** [comment]
Thank you for your advice!
I will try again using the parameters you recommended.
-Taeksoo
